src/components/Popover.vue

```diff
@@ -65,24 +65,23 @@ export default class Popover extends Vue {
   parent: HTMLElement | null = null;
   parents: HTMLElement[] = [];
 
-  get isBottom() {
-    return this.bottom;
+  beforeDestroy() {
+    if (this.visible) {
+      this.destroyPositioning();
+    }
```

Comment:
:100: 
We don't enforce positioning rules of vue options in this repo. Nonetheless, it ease the read to order events following their order in the lifecycle of the component.
https://vuejs.org/v2/style-guide/#Component-instance-options-order-recommended